src/PrestaShopBundle/Controller/Admin/ErrorController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * 2007-2019 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Controller\Admin;
+
+use PrestaShop\PrestaShop\Core\Domain\Configuration\Command\SwitchDebugModeCommand;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Manages Error pages (e.g. 500)
+ */
+class ErrorController extends FrameworkBundleAdminController
+{
+    /**
+     * Enables debug mode from error page (500 for example)
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function enableDebugModeAction(Request $request)
```

Comment:
I think an `@AdminSecurity` is mandatory here

Reply:
Hmmm, this is interesting case. `@AdminSecurity` works by using legacy controllers, but this is new controller that does not have equivalent controller in legacy.

Reply:
Maybe we could require the same access rights than the ones for the Maintenance page where people can enable/disable debug mode ? I think it would make sense

For the record, if we create new pages, it's not an issue not to have a legacy controller. We can create fake entries in the tab SQL table to be used for the new page.

Reply:
> We can create fake entries in the tab SQL table to be used for the new page.

Yeah, but then we need also need update "Permissions" page, right? I'll use "Performance" page as it is the place where debug mode can be configured.